test/issues/96-param-names.js

```diff
@@ -26,32 +26,40 @@
     }, function (a, b) {
```

Comment:
Can you get these variables to match in name with those tests below or vise versa?

Reply:
not sure what you mean?